src/main/java/com/company/Main.java

```diff
@@ -1,29 +1,18 @@
 package com.company;
 
-import java.util.Scanner;
+import java.io.File;
 
 public class Main {
     public static void main(String[] args) {
 
-        System.out.println("write name of file: ");
-        Scanner scanner = new Scanner(System.in);
-        String nameOfFile = scanner.nextLine();
+        Counter counter = new Counter();
+        int numberOfLines = counter.countLines(new File("/Users/mzyrkowski/IdeaProjects/Ważne - zadania/007_Word_Counter_New/try.txt"));
```

Comment:
dalej masz hardcodowane sciezki. To tak nie moze zostac ani tu, ani w testach. 
W programie uzytkownik ma podac albo nazwe pliku albo sciezki do pliku. W testach musisz to zrobic tak zeby sobie same znajdowaly odpowiednie pliki.